.github/labels.yml

```diff
@@ -0,0 +1,789 @@
+- color: FFCAD4
```

Comment:
all labels are the same as what we have, this is crawled by `label-exporter` https://github.com/micnncim/label-exporter

We can start adding/removing/sorting/grouping/organizing the labels in future PRs